src/status/wallet.nim

```diff
@@ -186,7 +186,8 @@ proc generateNewAccount*(self: WalletModel, password: string, accountName: strin
 
 proc addAccountsFromSeed*(self: WalletModel, seed: string, password: string, accountName: string, color: string): string =
   let mnemonic = replace(seed, ',', ' ')
-  let generatedAccount = status_accounts.multiAccountImportMnemonic(mnemonic)
+  var generatedAccount = status_accounts.multiAccountImportMnemonic(mnemonic)
+  generatedAccount.derived = status_accounts.deriveAccounts(generatedAccount.id)
```

Comment:
The only reason we're attaching `derived` here, is because `saveAccount()` relies on that to return a `DerivedAccount` which needs the correct `address` ("correct" being the default account):

https://github.com/status-im/nim-status-client/blob/68aafaba6fdee1521bdf6075f35b1e6b75a2f2a6/src/status/libstatus/accounts.nim#L244-L262